data/moves.ts

```diff
@@ -16458,13 +16458,13 @@ export const Moves: {[moveid: string]: MoveData} = {
 				this.add('-fail', source, 'move: Shed Tail');
 				return this.NOT_FAIL;
 			}
-			if (source.hp <= source.maxhp / 2 || source.maxhp === 1) { // Shedinja clause
+			if (source.hp <= (source.maxhp + 1) / 2) { // rounds up
```

Comment:
```suggestion
			if (source.hp <= Math.ceil(source.maxhp / 2) || source.maxhp === 1) { // Shedinja clause
```

Reply:
Turned the +1 stuff (I was thinking of integer division) to Math.ceil, but I think the shedinja clause should not return.